test_requirements.txt

```diff
@@ -0,0 +1,21 @@
+coverage==4.1
+Flask==0.11.1
+Flask-SQLAlchemy==2.1
+itsdangerous==0.24
+Jinja2==2.8
+jsonschema==2.5.1
+MarkupSafe==0.23
+pluggy==0.3.1
+psycopg2==2.6.1
+py==1.4.31
+pygraphviz==1.3.1
+pytest==2.9.2
+SQLAlchemy==1.0.13
+tox==2.3.1
+virtualenv==15.0.2
+Werkzeug==0.11.10
+wheel==0.29.0
+
+# Tests also require a local postgres database with a schema named test in a
```

Comment:
I don't think that it's pertinent to put this comment in the requirement.txt

We can put that in the readme.